README.md

```diff
@@ -1,2 +1,70 @@
-# ballerina-openapi
-Ballerina OpenApi-Tool
+# Ballerina-OpenAPI
```

Comment:
Let's get this reviewed by someone from the doc team when merging to the master branch.